pilot/pkg/networking/core/listener_waypoint.go

```diff
@@ -1020,6 +1065,10 @@ func buildCommonConnectTLSContext(proxy *model.Proxy, push *model.PushContext) *
 		TlsMaximumProtocolVersion: tls.TlsParameters_TLSv1_3,
 		TlsMinimumProtocolVersion: tls.TlsParameters_TLSv1_3,
 	}
+	if isEastWestGateway(proxy) {
+		// The gateway should accept TLS 1.2 for compatibility with older clients.
+		ctx.TlsParams.TlsMinimumProtocolVersion = tls.TlsParameters_TLSv1_2
```

Comment:
What older clients are we supporting?

Reply:
This should probably read: "clients using older versions of TLS". Not every client uses TLS 1.3 yet (it's just recently FIPS compliant); we could get away with this with waypoints

Reply:
Aren't all clients still programmed by Istiod?

Reply:
Ah you know what, the time I ran into this was when I was testing with the echo client and wireshark. Not needed anymore

Reply:
E/W should be only accessed directly by ztunnel

Reply:
Yep, sidecar, ztunnel or egress gateway